frontend/annotation_resolver.py

```diff
@@ -161,19 +163,19 @@ def add_annotated_function_axioms(self, args_types, solver, annotated_function,
         args_annotations = annotated_function.args_annotations
         result_annotation = annotated_function.return_annotation
 
+        axioms = []
+
         if len(args_types) != len(args_annotations):
-            raise TypeError("The function expects {} arguments. {} were given.".format(len(args_annotations),
-                                                                                       len(args_types)))
+            return None
```

Comment:
The reason is that we are not sure whether it's actually a built-in call or not. For example, if you have a user-defined method whose name matches the name of a built in method but whose number of arguments doesn't.

Example:
```python
class A:
    def append(self, e1, e2):
         pass

x = [1,2,3]
y = A()

x.append(1)
y.append(1, 2)
```

So the only way we will add no axioms is when the code contains a function call with invalid number of arguments.